lazy_theta_star_p_planner/global_planner_plugin.xml

```diff
@@ -0,0 +1,5 @@
+<library path="lazy_theta_star_p_planner_plugin">
+    <class name="lazyThetaStarP_Planner/LazyThetaStarP_Planner" type="lazyThetaStarP_Planner::LazyThetaStarP_Planner" base_class_type="nav2_core::GlobalPlanner">
```

Comment:
We'll need to work on this name :smile: 

Lazy Theta Star not enough? I think the `P` variation is something for documentation README / website but a little TMI for the name

Reply:
Also add a 10 word description, doesn't need to be alot, just not empty